bisection/application.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python2
+
+"""The fake-core binary of your application
+
+Assume this is the core binary of your application. Assume your goal
+is to keep this binary running without any problems. The purpose is
+to train you using `git bisection` to find out which commit causes
+this binary stopped to work.
+
+The algorithm of this script is to find out how many lines of
+comments are there in the repo. If it exceeds the number we pass
+to this program, it'll raise an exception.
+"""
+
+from __future__ import print_function
+
+import argparse
+import os
+import sys
+
+COMMENT_STYLES = {
+    '.java': ['//', ('/*', '*/')],
```

Comment:
('/**', '*/') should also capture javadocs

Reply:
Just to be clear: only ('/**', '*/') is valid in Java or ('/*', '*/') is also valid?

Reply:
/** */ is used specifically for documentation e.g. class , method, or field docs.

/* */ just denotes a multi-line comment that can appear within the body of a method.

Reply:
Thanks for the clarification. If both are valid, then the current tuple ('/\*', '\*/') should be enough, as I only check the starting of the line. (If add two, javadocs will be counted twice)